mkdocs.yml

```diff
@@ -1,5 +1,7 @@
 site_name: Narwhals
 repo_url: https://github.com/narwhals-dev/narwhals.git
+watch:
+  - narwhals
```

Comment:
ooh, what's this?

Reply:
Sorry it slipped in, it is for dynamic reloading if something is in the folder changes